src/storage/CookiesStorage.js

```diff
@@ -67,21 +67,67 @@ class CookiesStorage {
     if (document.cookie === oldCookies) {
       return null;
     }
-    
+
     // Since the cookie was updated with a dummy value, remove it by setting
     // expiry to a date in the past.
-    const pastDate = "Thu, 01 Jan 1970 00:00:00 GMT";
-    document.cookie = 
+    const pastDate = 'Thu, 01 Jan 1970 00:00:00 GMT';
+    document.cookie =
       `registrable_domain=;domain=${domain};expires=${pastDate}`;
 
     return domain;
   }
 
   /** @override */
-  save(key, value, secondsToLive) {}
+  save(key, value, secondsToLive =
```

Comment:
Should this method have a jsdoc? I don't remember the guidance on overrides, please double-check that no jsdoc is necessary (I guess I'm not following why API now redeclares secondsToLive as  parameter).